pytripgui/main_window_qt_vc/main_window_view.py

```diff
@@ -35,8 +35,15 @@ def browse_file_path(self, name, extension, path=None):
         """
         :return full file path, or empty string
         """
-        selected_file = QFileDialog.getOpenFileName(self.ui, name, path, extension)
-        return selected_file[0]
+        selected_file_path, selected_file_extension = QFileDialog.getOpenFileName(self.ui, name, path, extension)
+        return selected_file_path
+
+    def save_file_path(self, caption, extension, path=None):
+        """
+        :return full file, or empty string
```

Comment:
What is full file ? File full of bytes or a path ? Is it absolute path, relative path ?

Reply:
changed to "full file path (absolute)"